.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    target-branch: "master"
+    schedule:
+      interval: "daily"
```

Comment:
Perhaps this should be weekly just to manage the number of PRs?

Reply:
The [default max pull requests](https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#open-pull-requests-limit) for dependabot is 5, which I think is manageable.